Quick fix for deleting labels in RC >= 1.1

binsentsu opened the pull request:
Fix for deleting labels in Roundcube >= 1.1

binsentsu committed:
Quick fix for deleting labels in RC >= 1.1

mckaygerhard commented:
are this fix still compatible with older releases?? works also for 0.9 and 1.0?

mckaygerhard commented on message_label.js:
```diff
@@ -46,6 +46,31 @@ rcube_webmail.prototype.unlabel_messages = function(row, label, type) {
     }
     $(remove).parent().remove();
 
+    a_uids = [];
```
this need php 5.4 , must be changed to a_uids = array();

Oygron committed:
Update README

Adaptation à ce répo

PascalSchmidt committed:
compatibilité RC1.2

PascalSchmidt committed:
Ajout du fichier composer.json

PascalSchmidt committed:
Brackets uniformisation + tabs

zewish committed:
Add Bulgarian translation

Oygron committed:
Merge pull request #5 from zewish/master

Add Bulgarian translation

alegag committed:
Some PSR-2 fixes.

alegag committed:
Support multiple skins (not only Larry).

alegag committed:
Using a placeholder instead of the "input" label in settings (matches)

Also changed a French label to better describe what does the field (label_matches).

alegag committed:
Changed commented "write_log()" to "rcube::write_log()" for compatibility with new RoundCube versions.

alegag committed:
Fixed a variable accidental overwrite and compatibility with latest RoundCube version.

alegag committed:
Added "flagged" icon in Settings menu.

Oygron committed:
Merge pull request #7 from alegag/master

Series of changes for compatibilty with RoundCube 1.3.4

binsentsu committed:
Merge pull request #1 from Oygron/master

pull changes back

binsentsu committed:
Make Compatible with RC 1.4 Elastic Skin

binsentsu committed:
Correct meta info

mckaygerhard commented:
seems there's no rc 1.1.1 compatbility
